src/components/PageTitle/index.tsx

```diff
@@ -12,22 +12,24 @@ type Props = {
   smaller?: boolean
   subTitle?: string
   title: string
+  shouldAutoFocus?: boolean
 }
 
 const PageTitle: FunctionComponent<Props> = ({
   title,
   subTitle,
   smaller,
   className,
+  shouldAutoFocus = true,
 }) => {
   const isFullScreen = useSelector<RootState, boolean | undefined>(
     (state) => state.globals.isFullScreen
   )
   const containerRef = useRef<HTMLSpanElement>(null)
 
   useEffect(() => {
-    containerRef.current && containerRef.current.focus()
-  }, [title, subTitle])
+    containerRef.current && shouldAutoFocus && containerRef.current.focus()
+  }, [title, subTitle, shouldAutoFocus, isFullScreen])
```

Comment:
I'd love to have an extra comment here to explain why `isFullScreen` needs to be in the dependencies list

Reply:
Ah, that's my mistake. I was initially trying to use `isFullScreen` to determine if it should be auto focused on but that wasn't the right conditional to use. Will remove it from the dependencies list.